addons/planner/static/src/js/planner.js

```diff
@@ -96,6 +96,7 @@
             'click .oe_planner div[id^="planner_page"] a[href^="#planner_page"]': 'change_page',
             'click .oe_planner li a[href^="#planner_page"]': 'change_page',
             'click .oe_planner div[id^="planner_page"] button[data-pageid^="planner_page"]': 'mark_as_done',
+            'hide.bs.modal':'on_close'
```

Comment:
If necessary, those changes should be done in a specific commit / branch. What is the related bug / feature ?